app/src/main/java/com/mobiledevelopmentfoodapp/dto/Order.kt

```diff
@@ -4,7 +4,11 @@ package com.mobiledevelopmentfoodapp.dto
 //import androidx.room.PrimaryKey
 import com.google.gson.annotations.SerializedName
 
-data class Order(var customerId: Int, var name: String, var products: ArrayList<Food>, var totalPrice: Double, @SerializedName("id") var orderId:Int = 0 ) {
+data class Order(var customerId: Int,
+                 var name: String,
+                 var products: ArrayList<Food>,
+                 var totalPrice: Double,
+                 @SerializedName("id") var orderId:Int = 0 ,) {
```

Comment:
Added in trailing commas to simplify code generation according to https://kotlinlang.org/docs/coding-conventions.html#trailing-commas.

Reply:
Changed the formatting of this class header according to https://kotlinlang.org/docs/coding-conventions.html#class-headers.